packages/as-types/assembly/__tests__/arrays.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { staticArrayToUint8Array } from '../serialization';
+
+describe('arrays', () => {
+  test('staticArrayToUint8Array', () => {
+    const myArray = StaticArray.fromArray<u8>([0, 1, 2, 3, 4]);
+    const result = staticArrayToUint8Array(myArray);
+
+    expect(result instanceof Uint8Array).toBe(true);
+    expect(result.length).toBe(myArray.length);
+    for (let i = 0; i < myArray.length; i++) {
+      expect(result[i]).toBe(myArray[i]);
+    }
+  });
```

Comment:
I would add a etst to confirm that modification is possible:
```suggestion
    typedArray[0] = 10;
    expect(typedArray[0]).toBe(<u8>10);
```

Reply:
sure